dags/benk/environment.py

```diff
@@ -0,0 +1,45 @@
+from typing import List
+
+from airflow.kubernetes.secret import Secret
+from airflow.models import Variable
+from kubernetes.client import V1EnvVar
+
+
+class OperatorEnvironment:
+    """Base environment class to put environment variables on.
+
+    TODO: Use 'Secret' object for passwords instead.
+    """
+    def env_vars(self) -> List[V1EnvVar]:
+        """Returns all env vars in this object."""
+
+        def _is_env_var(k) -> bool:
```

Comment:
Maybe a warning in case of invalid env var?

Reply:
nah; that would cause a lot of warnings. Maybe just check if it is a Variable instead.

Reply:
Thats even better indeed, if we only use Variables there

Reply:
impossible though; as the values are returned at class creation time, not object creation time.